osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+logger = logging.getLogger(__name__)
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def delete_tasks_results(tasks_to_delete):
+    from kobo.hub import models
+    for task in tasks_to_delete:
+        try:
+            path_to_delete = models.Task.get_task_dir(task.id)
+            shutil.rmtree(path_to_delete)
+        except OSError as e:
+            logger.error("Error deleting tasks folder: %s", e)
```

Comment:
Do we record which tasks (in fact task-specific directories) have already been deleted?

If not we will hit `FileNotFoundError` exceptions for the already deleted tasks on each subsequent run.  It is also wasteful to iterate over 200000 directories each day, just to discover that they do not exist any more.  I think we should introduce a task-specific flag to mark the tasks with deleted data and exclude them in the initial db query already.